subsys/bluetooth/mesh/transport.c

```diff
@@ -1077,8 +1123,7 @@ int bt_mesh_ctl_send(struct bt_mesh_net_tx *tx, uint8_t ctl_op, void *data,
 
 	tx->ctx->app_idx = BT_MESH_KEY_UNUSED;
 
-	if (tx->ctx->addr == BT_MESH_ADDR_UNASSIGNED ||
-	    BT_MESH_ADDR_IS_VIRTUAL(tx->ctx->addr)) {
+	if (tx->ctx->addr == BT_MESH_ADDR_UNASSIGNED) {
```

Comment:
@omkar3141 

I'm skeptical about the modifications here.

<img width="815" alt="image" src="https://github.com/zephyrproject-rtos/zephyr/assets/26378606/de7099c5-655d-4f1d-b3dd-8499b0a8454c">

**AND**

<img width="923" alt="image" src="https://github.com/zephyrproject-rtos/zephyr/assets/26378606/863057c8-d995-42f4-80fe-1c32f89d655d">

<img width="913" alt="image" src="https://github.com/zephyrproject-rtos/zephyr/assets/26378606/fcf8aa47-634e-44e4-98c7-9adffc851b43">

The spec clearly requires that virtual address cannot be used in the DST field of CTL message, but the DST of path echo request may be a virtual address.

Reply:
Thanks for pointing out. Seems like spec has some inconsistency in two places.

Reply:
We will fix this transport layer bug seperately.